packaging/installer/kickstart.sh

```diff
@@ -138,7 +138,7 @@ telemetry_event() {
   if [ -f /etc/machine-id ]; then
     DISTINCT_ID="$(cat /etc/machine-id)"
   elif command -v uuidgen > /dev/null 2>&1; then
-    DISTINCT_ID="$(uuidgen)"
+    DISTINCT_ID="$(uuidgen | tr '[:upper:]' '[:lower:]')"
```

Comment:
Strictly speaking, the Cloud never sees this value, as it0s only used for analytics reporting.

However, normalizing things here is a good idea too, so we should just keep this change.